reporter/src/main/kotlin/StatisticsCalculator.kt

```diff
@@ -80,12 +71,8 @@ object StatisticsCalculator {
         )
     }
 
-    private fun getOpenIssues(
-        ortResult: OrtResult,
-        resolutionProvider: ResolutionProvider,
-        ortConfig: OrtConfiguration
-    ): IssueStatistics {
-        val openIssues = ortResult.getOpenIssues(Severity.HINT).filterNot { resolutionProvider.isResolved(it) }
+    private fun getOpenIssues(ortResult: OrtResult, ortConfig: OrtConfiguration): IssueStatistics {
+        val openIssues = ortResult.getOpenIssues(Severity.HINT)
```

Comment:
Note (to myself as well): The signature of `getOpenIssues()` is a bit inconsistent with `getRuleViolations()` and `getVulnerabilities()` in that it offers no `omitResolved` parameter, but always filters for unresolved issues.

As such, the `.filterNot { resolutionProvider.isResolved(it) }` call could have been dropped without a change of functionality independently of this commit / PR. Maybe this should be clarified in the commit message. Or do we even take the chance and align the API as part of this PR?

Reply:
> As such, the .filterNot { resolutionProvider.isResolved(it) } call could have been dropped without a change of functionality independently of this commit / PR. 

Can you help understanding _why_ it would not change functionality?

Reply:
`getOpenIssues()` (as opposed to `getIssues()`) is implemented like this:

https://github.com/oss-review-toolkit/ort/blob/07f9efb09370a1936f69a35f181d1155f347f0fb/model/src/main/kotlin/OrtResult.kt#L270-L276

so the `getResolutions().issues.none { it.matches(issue) }` part already only keeps issues for which no resolution matches. Which means the `.filterNot { resolutionProvider.isResolved(it) }` never was necessary here, if I'm not mistaken.

Reply:
Thanks!

Reply:
> As such, the .filterNot { resolutionProvider.isResolved(it) } call could have been dropped without a change of functionality independently of this commit / PR. Maybe this should be clarified in the commit message. Or do we even take the chance and align the API as part of this PR?

However, this comment assumes that `OrtResult` contains all resolutions. This wasn't true until my recent changed which changed the reporter command to ensure that. And that change was a preparation for this commit, so to me it makes sense to remove it in this commit. Does this make sense to you now as well ?